src/Schema/Event.ts

```diff
@@ -93,3 +105,203 @@ export interface SuccessfullyLoggedIn {
+}
+
+/**
+ * A user taps a grouping of auctions on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedAuctionGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedAuctionGroup"
+ *    context_module: "auctionsRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Sale
+ *    destination_screen_owner_id: "5e95b37a2fdcb20012a0e082"
+ *    destination_screen_owner_slug: "forum-auctions-colour-theory-4"
+ *    horizontal_slide_position: 3
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedAuctionGroup {
+  action: ActionType.tappedAuctionGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
+  type?: "thumbnail" | "header" | "stub"
+}
+
+/**
+ * A user taps a grouping of collections on iOS
+ *
+ * This schema describes events sent to Segment from [[tappedCollectionGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedCollectionGroup"
+ *    context_module: "collectionRail"
+ *    context_screen_owner_type: Home
+ *    destination_screen_owner_type: Collection
+ *    destination_screen_owner_slug: "limited-edition-prints-trending-artists"
+ *    horizontal_slide_position: 2
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedCollectionGroup {
+  action: ActionType.tappedCollectionGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
+  type: "thumbnail" | "header" | "stub"
+}
+
+/**
+ * A user taps a grouped explore module (City Guide, for example)
+ *
+ * This schema describes events sent to Segment from [[tappedExploreGroup]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "tappedExploreGroup"
+ *    context_module: "cityGuideCard"
+ *    context_screen_owner_type: Search
+ *    destination_screen_owner_type: CityGuideGuide
+ *    module_height: "double"
+ *    type: "thumbnail"
+ *  }
+ * ```
+ */
+export interface tappedExploreGroup {
+  action: ActionType.tappedExploreGroup
+  context_module: ContextModule
+  context_screen_owner_type: ScreenOwnerType
+  context_screen_owner_id?: string
+  context_screen_owner_slug?: string
+  destination_screen_owner_type: ScreenOwnerType
+  destination_screen_owner_id?: string
+  destination_screen_owner_slug?: string
+  horizontal_slide_position?: number
+  module_height?: "single" | "double"
```

Comment:
I wonder why `module_height` and `type` are optional properties on some but not all of these interfaces?

Reply:
inadvertent and fixed! they should be optional everywhere. the only required properties should be `context_module`, `context_screen_owner_type`, and `destination_screen_owner_type`